app/models/ext_management_system.rb

```diff
@@ -622,11 +622,33 @@ def destroy_queue
     task.id
   end
 
+  def ems_workers
+    MiqWorker.find_alive.where(:queue_name => queue_name)
+  end
+
+  def wait_for_ems_workers_removal
+    return if Rails.env.test?
```

Comment:
I'm not sure how else to test this if this method will loop and wait for the worker rows to be removed.

Reply:
I wonder if we could stub the `#kill_async` and have it execute it directly instead of putting test specifics in the main code...might play with this later

Reply:
Yeah, that's a possibility.  The problems I had in the 2 `ext_management_system_spec.rb`  examples changed in this PR:

* I'd need to stub `#kill_async` to delete the rows
* I'd need to either stub using `any_instance` since `destroy` queues for the Ems and deliver would get a different Ems unless you `any_instance` or you'd need to stub deliver to get your specific Ems instance with the stubbed method.